app/frontend/src/features/donations/DonationsBox.tsx

```diff
@@ -0,0 +1,474 @@
+      border: `2px solid ${theme.palette.primary.main}`,
+      backgroundColor: theme.palette.background.paper,
+    },
+  },
+
+  buttonSecondaryActive: {
+    "&&": {
+      border: `2px solid ${theme.palette.primary.main}`,
+      backgroundColor: theme.palette.background.paper,
+    },
+  },
+
+  buttonMain: {
+    backgroundColor: theme.palette.primary.main,
+    "&&": {
+      borderRadius: "0.5rem",
+      boxShadow: "initial",
+    },
+    "&:hover": {
+      opacity: 0.4,
+      backgroundColor: theme.palette.primary.main,
+    },
+    "& .MuiButton-label": {
+      color: theme.palette.background.paper,
+      transition: `color ${theme.transitions.duration.short} ${theme.transitions.easing.easeInOut}`,
+    },
+  },
+
+  buttonMainText: {
+    color: theme.palette.background.paper,
+    fontWeight: 700,
+    transition: `color ${theme.transitions.duration.short} ${theme.transitions.easing.easeInOut}`,
+  },
+
+  formGroup: {
+    width: "100%",
+  },
+
+  marginY2: {
+    margin: theme.spacing(2, 0),
+  },
+
+  marginBottom2: {
+    marginBottom: theme.spacing(2),
+  },
+
+  inputWrapper: {
+    position: "relative",
+    "&::before": {
+      content: `'${DONATIONSBOX_CURRENCY}'`,
+      position: "absolute",
+      top: "50%",
+      transform: "translateY(-50%)",
+      left: theme.spacing(1),
+      color: theme.palette.grey[600],
+      fontWeight: 700,
+      fontSize: theme.typography.button.fontSize,
+    },
+  },
+
+  inputNumber: {
+    width: "100%",
+    height: "100%",
+    border: `2px solid ${theme.palette.grey[200]}`,
+    paddingLeft: theme.spacing(2),
+    color: theme.palette.grey[600],
+    fontWeight: 700,
+    fontSize: theme.typography.button.fontSize,
+    "&&": {
+      borderRadius: theme.shape.borderRadius * 2,
+      boxShadow: "initial",
+    },
+    "&:hover": {
+      border: `2px solid ${theme.palette.primary.main}`,
+    },
+    "&:focus-visible": {
+      border: `2px solid ${theme.palette.primary.main}`,
+      outline: "none",
+      boxShadow: "none",
+    },
+  },
+
+  inputNumberActive: {
+    "&&": {
+      border: `2px solid ${theme.palette.primary.main}`,
+    },
+  },
+}));
+
+export interface DonationFormData {
+  amount: number;
+  recurring: boolean;
+}
+
+export default function DonationsBoxMixed() {
+  const stripePromise = loadStripe(process.env.REACT_APP_STRIPE_KEY!);
+
+  const classes = useStyles();
+
+  const [success, setSuccess] = useState(false);
+  const [cancelled, setCancelled] = useState(false);
+
+  const location = useLocation();
+
+  const [isAmount, setIsAmount] = useState(false);
+
+  useEffect(() => {
+    const query = new URLSearchParams(location.search);
+    setSuccess(query.get("success") ? true : false);
+    setCancelled(query.get("cancelled") ? true : false);
+  }, [location]);
+
+  const {
+    control,
+    handleSubmit,
+    reset: resetForm,
+    errors,
+  } = useForm<DonationFormData>({ defaultValues: { recurring: true } });
```

Comment:
```suggestion
  } = useForm<DonationFormData>();
```
You've already set the default value on the controller